source/content/quicksilver.md

```diff
@@ -63,23 +63,24 @@ You can hook into the following workflows:
 
 | Workflow                               | Description                                                         | Location of webphp runtime | Notes                                       |
 |:-------------------------------------- |:------------------------------------------------------------------- |:-------------------------- |:------------------------------------------- |
-| `clear_cache`                          | Clear CMS and Edge Cache                                            |                            |                                             |
-| `clone_database`                       | Clone database between environments                                 | target (to_env)            |                                             |
-| `deploy`                               | Deploy code to Test or Live                                         | target                     |                                             |
-| `deploy_product`                       | Create site                                                         | Dev                        | `after` stage valid, `before` stage invalid 
-| `sync_code`                            | Use the command `git push` to change the code; commit via the Pantheon Dashboard, upstream update, or Multidev merge | Dev or Multidev            |                                             |
-| `create_cloud_development_environment` | Create Multidev environment                                         | Multidev                   | `after` stage valid, `before` stage invalid |
-| `autopilot_vrt`                        | Autopilot Visual Regression test                                    | "Autopilot" Multidev       | `after` stage valid, `before` stage invalid |
+| `autopilot_vrt`                          | Autopilot Visual Regression test                                             | "Autopilot" Multidev                            | `after` stage valid, `before` stage invalid                                           |
+| `clear_cache`                       | Clear CMS and Edge Cache                                 |             |                                             |
+| `clone_database`                               | Clone database between environments                                         | target (to_env)                    |                                             |
+| `create_cloud_development_environment`                       | Create Multidev environment                                             | Multidev               | `after` stage valid, `before` stage invalid
+| `deploy`                            | Deploy code to Test or Live | Dev or Multidev            | target                                            |
```

Comment:
One small correction... the `deploy` hook "Location of webphp runtime" should be "target" like it is on the live site.. in these changes "target" got shifted into the "Notes" section.  Whitney already confirmed she'll make that correction, so I'm approving.